pipeline/activators/aviris-l2/main.py

```diff
@@ -0,0 +1,316 @@
+import argparse
+from contextlib import contextmanager
+import ftplib
+import json
+from math import floor
+import os
+from pathlib import Path
+import shutil
+import sys
+import tarfile
+from tempfile import mkdtemp
+import threading
+from time import time
+from urllib.parse import urlparse, urlunparse
+
+import boto3
+from boto3.s3.transfer import TransferConfig
+from osgeo import gdal
+import pystac
+import requests
+
+
+GB = 1024 ** 3
+s3 = boto3.client("s3")
+
+AVIRIS_ARCHIVE_COLLECTION_ID = "aviris-collection"
```

Comment:
Well the question was more about consistency 🤷

Reply:
@CloudNiner what would be the benefits of a single self contained py script? I see no issues we having 2 or 3 files in the same folder. 

You mean it is a regular py pratice to have a single py script?

Reply:
I'm not sure there is a regular python practice on this, specifically in the case when nothing in the script is a dependency for other code outside the script or the script isn't part of a larger project.

Warning -- this is a personal preference -- I just find it easier, when a script is entirely self contained like this one and in the absence of knowledge of a python best practice, to just keep everything in the same file. So that's how I wrote it here. For everything not in `main()` it's just as easy to scroll past it as it is to look at it in a separate file.